modules/rostests/apitests/include/apitest.h

```diff
@@ -8,12 +8,15 @@
 #include <wine/test.h>
 #undef WIN32_NO_STATUS
 
-#include <pseh/pseh2.h>
+/* Do not include if not needed */
```

Comment:
This comment doesn't seem to have any context here.
Also, why can't we always include the header? If the compiler supports `__try`, all the header should need to do is `#define __SEH2_TRY __try` and such (which it already does for MSVC).

Reply:
Right. I'll rephrase the comment. It's more : do not include if not available.
The PSEH headers are not universally available anymore, forcing anyone who want to use them to actually link their library/module/executable against the pseh library. This is intended. Linking to pseh also yields the __USE_PSEH2__ definition to avoid such situations where the header might need PSEH, but then it's up to the includer to actually use it.

I added the __try versions for two reasons :
  - (weak reason) there is no need for MSVC to use PSEH2 macros (__USE_PSEH2__ is not defined there)
  - (strong reason) Having an undefined __try makes it easier for us to think "oh right I forgot pseh" than having to wonder where this undefined macro comes from.